homeassistant/components/doorbird.py

```diff
@@ -24,14 +25,20 @@
 CONF_DOORBELL_EVENTS = 'doorbell_events'
 CONF_CUSTOM_URL = 'hass_url_override'
 
+DEVICE_SCHEMA = vol.Schema({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_DOORBELL_EVENTS): cv.boolean,
+    vol.Optional(CONF_CUSTOM_URL): cv.string,
+    vol.Optional(CONF_NAME): cv.string
+})
+
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
-        vol.Required(CONF_HOST): cv.string,
```

Comment:
I would like to suggest this schema:

```
doorbird:
  gateways:
    - host: <host_ip>
      username: <username>
      password: <password>
      doorbell_events: 1
      name: Side Entry
    - host: <host_ip>
      username: <username>
      password: <password>
      doorbell_events: 1
      name: Front Door
```
In this case the schema can be extended in future with parameters applied to all gateways without another beaking change of the configuration.

cp. https://github.com/home-assistant/home-assistant/pull/13517

Reply:
This schema is better (extensible)
Although these are not really gateways I think. So `gateways` should be `devices` probably.